features/cli/deploy.feature

```diff
@@ -363,7 +363,16 @@ Feature: deployment related features
       | name     | ab-example-a     |
       | replicas | 0                |
     Then the step should succeed
-    Given I wait until number of replicas match "0" for replicationController "ab-example-a-1"
+    Given number of replicas of "ab-example-a" deployment config becomes:
+      | desired   | 0 |
+      | current   | 0 |
+      | updated   | 0 |
+      | available | 0 |
+    Given number of replicas of "ab-example-b" deployment config becomes:
+      | desired   | 1 |
+      | current   | 1 |
+      | updated   | 1 |
+      | available | 1 |
```

Comment:
@zhouying7780 do we need steps from 371 to 375 ?  I understand that we here are making sure to have replicas as '0' right.